src/Providers/DataGrid/Wijmo/Helper/CellTemplateFactory.ts

```diff
@@ -10,11 +10,16 @@ namespace Providers.DataGrid.Wijmo.Helper.CellTemplateFactory {
         type: OSFramework.DataGrid.Enum.CellTemplateElementType,
         binding: string,
         callback: (item) => void,
-        altText?: string
+        altText?: string,
+        externalURL?: string
     ): wijmo.grid.ICellTemplateFunction {
         let cellTemplate: wijmo.grid.ICellTemplateFunction;
 
         const hasFixedText = binding.startsWith('$');
+        const hasExternalURL = externalURL?.substring(0, 4) === 'http';
```

Comment:
I don't know if this validation is enough, since nowadays we can use links without the http or https or even without the www... Why we didn't just check if externalURL it's filled?